spec/controllers/verify/sessions_controller_spec.rb

```diff
@@ -191,6 +191,20 @@
           expect(response).to redirect_to verify_fail_url
         end
       end
+
+      context 'attempt window has expired, previous attempts == max-1' do
+        before do
+          user.idv_attempts = 2
```

Comment:
would it make sense to reference some type of env var or constant here? Assuming this is supposed to be "max -1" but could be clearer if we ref'd a variable.